site/en/blog/how-fugu-is-my-browser-and-how-fugu-is-the-web/index.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: 'layouts/blog-post.njk'
+title: '"How Fugu is my browser?" and "How Fugu is the web?"'
+subhead: >
+  To answer these questions, check out a browser testing site and a browser extension, both focused
+  on the web's capabilities.
+date: 2023-01-16
+# updated: 2023-01-16
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/8FZcBmFowbDKWxpkOytx.jpg
+alt: Blowfish swarm swimming in the ocean.
+authors:
+  - thomassteiner
+tags:
+  - capabilities
+---
+
+## How Fugu is my browser?
+
+Many developers and executives alike don't realize how capable the modern web has become.
+Double-clicking an image file so it opens in an associated PWA, making modifications, saving the
+changes back to the file, and then copying the image contents over into another app or sharing it to
+an email client, is a flow that wasn't possible on the web until recently. But APIs developed in the
+context of [Project Fugu](/capabilities/) 🐡 such as [File Handling](/articles/file-handling/),
+[File System Access](/articles/file-system-access/),
+[Async Clipboard](https://web.dev/async-clipboard/), and [Web Share](https://web.dev/web-share/)
+have made this flow possible.
+
+With this in mind, I have developed a test application with the tongue-in-cheek name
+[How Fugu is my browser?](https://howfuguismybrowser.dev/) that you can take for a spin to see what
+Project Fugu APIs your browser of choice supports. Since not all features are exposed on all
+platforms—for example, the [Contact Picker API](/articles/contact-picker/) is currently only exposed
+on mobile—it's technically impossible to reach a score of 100% if you test on desktop (and vice
+versa), so regard it as more like a playful competition than absolute science. For each tested
+feature, there's a link to the relevant documentation so you can learn more about the feature. Where
+feature detection is possible, there is also a note on whether the feature is supported by your
+browser or not, and finally
+[page load statistics linked to Chrome Status](https://chromestatus.com/metrics/feature/timeline/popularity)
+that tell you how popular over time a given feature is.
```

Comment:
```suggestion
To learn what is possible, check out the application [How Fugu is my browser?](https://howfuguismybrowser.dev/) and see what Project Fugu APIs your browser of choice supports. Not all features are exposed on all
platforms—for example, the [Contact Picker API](/articles/contact-picker/) is currently only exposed
on mobile—therefore it's technically impossible to reach a score of 100% if you test on desktop (and vice
versa). Therefore, regard this test as a playful competition rather than absolute science. For each tested
feature, there's a link to the relevant documentation so you can learn more about the feature. Where
feature detection is possible, there is also a note on whether the feature is supported by your
browser or not, and finally
[page load statistics linked to Chrome Status](https://chromestatus.com/metrics/feature/timeline/popularity)
that tell you how popular over time a given feature is.
```